src/proxy/src/main/java/ee/ria/xroad/proxy/clientproxy/ClientProxy.java

```diff
@@ -251,7 +251,9 @@ private void createHandlers() throws Exception {
 
         final Slf4jRequestLogWriter writer = new Slf4jRequestLogWriter();
         writer.setLoggerName(getClass().getPackage().getName() + ".RequestLog");
-        final CustomRequestLog reqLog = new CustomRequestLog(writer, CustomRequestLog.EXTENDED_NCSA_FORMAT);
+
+        final CustomRequestLog reqLog = new CustomRequestLog(writer, "%{X-Forwarded-For}i "
```

Comment:
This is problematic due to "X-Forwarded-For" being an optional header that is not always present. Consider appending it instead, and (like other hearders) wrapping to quotes so that missing value is easier to detect. e.g.
```
       ... CustomRequestLog.EXTENDED_NCSA_FORMAT + " \"%{X-Forwarded-For}i\"";
```

Reply:
Changed.